packages/fast-components/src/slider-label/slider-label.template.ts

```diff
@@ -0,0 +1,28 @@
+import { html, when, ref } from "@microsoft/fast-element";
+import { SliderLabel } from "./slider-label";
+
+export const SliderLabelTemplate = html<SliderLabel>`
+    <div
+        ${ref("root")}
+        part="slider-label"
+        class="slider-label"
```

Comment:
does this need a prefix of slider? For part, the css selector would look like:

```
fast-slider::part(slider-label) {}
```

I think we can leave the prefix off here and just go with `label`:
```
fast-slider::part(label) {
    color: pink;
}
```

Reply:
I believe this has been changed in the code accordingly. part(label) will refer to just the label part of this component, all of what was "slider-label..." has been moved up into :host